pattern-1/bosh-release/jobs/wso2is_1/templates/config/datasources/bps-datasources.xml

```diff
@@ -31,7 +31,7 @@
             </jndiConfig>
             <definition type="RDBMS">
                 <configuration>
-                    <url>jdbc:mysql://<%=properties.wso2is.mysql.hostname%>:3306/<%=properties.wso2is.mysql.product_db%>?autoReconnect=true</url>
+                    <url>jdbc:mysql://<%=properties.wso2is.mysql.host%>:3306/<%=properties.wso2is.mysql.product_db%>?autoReconnect=true</url>
```

Comment:
Shall we go for a simplification of these keys, for example, we can have a data source connection url variable instead of all these fine grained key options, so that it's easy to template out in a common manner for all DB options ?

Reply:
The necessary refinements have been done in https://github.com/wso2/pivotal-cf-is/pull/44.